modules/create-storefront-x/index.js

```diff
@@ -103,8 +109,8 @@ async function getResponses() {
           message: 'Pick a demo application',
           choices: [
             { title: 'Blank application', value: 'blank' },
-            { title: 'Magento integration', value: 'magento' },
-            { title: 'Magento integration with additional modules (blog, brands, ...)', value: 'px' },
+            { title: 'Vanilla Magento integration', value: 'magento' },
```

Comment:
I know there are some more metions of Magento integration and Magento integration with additional modules in the docs. Could you comb through the docs and change those names too?